main.go

```diff
@@ -75,6 +76,7 @@ func init() {
 	_ = clientgoscheme.AddToScheme(scheme)
 	_ = infrav1.AddToScheme(scheme)
 	_ = infrav1exp.AddToScheme(scheme)
+	_ = infrav1expalpha.AddToScheme(scheme)
```

Comment:
Is this something we want to do whether or not the feature gate is enabled?

Reply:
I think we need this for the webhooks to work properly to forbid creating these resources.

Reply:
I suppose based on the way we're setting up webhooks and failing on feature gate not set means we're scaffolding these resources no matter what.

Reply:
jinx